docs/environment-variables.md

```diff
@@ -99,3 +99,4 @@ triggers in each request (defaults to 100000).
   [here](https://docs.rs/env_logger/0.6.0/env_logger/)
 - `THEGRAPH_STORE_POSTGRES_DIESEL_URL`: postgres instance used when running
   tests. Set to `postgresql://<DBUSER>:<DBPASSWORD>@<DBHOST>:<DBPORT>/<DBNAME>`
+- `GRAPH_ALLOW_UNRESPONSIVE`: If set, the process will not be killed if unresponsive.
```

Comment:
I would prefer `GRAPH_KILL_IF_UNRESPONSIVE` and requiring operators to manually set it.